wondrous-app/components/CreateEntity/CreateEntityModal/styles.tsx

```diff
@@ -702,12 +705,59 @@ export const CreateEntityAttachment = styled(ButtonUnstyled)`
   }
 `;
 
+export const CreateEntitySaveTemplateButton = styled(ButtonUnstyled)`
```

Comment:
palette aswell, also would probably be great if we move these to Template styles because this file is also reaching 1k lines and the component is more in the template picker scope